### PR TITLE
Tolerate all taints

### PR DESCRIPTION
**Pull request by corest**
Towards https://github.com/giantswarm/giantswarm/issues/6440
Add `GiantSwarmObservability` and `NodeWithImparedVolumes` tolerations

### CODE REVIEW

**Comment by corest** [comment]
Currently discussing with customers if they are fine to use predefined taints

**Comment by corest** [comment]
Customer is fine to adjust its taint with proposed from us. So the question is only about naming

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,10 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 - Fix systemd collector D-Bus connection. https://github.com/giantswarm/kubernetes-node-exporter/pull/44
 
+### Added
+
+- Added tolerations `GiantSwarmObservability=true` and `NodeWithImparedVolumes=true`.
```
Comment (kopiczko) [inline CHANGELOG.md]: This shouldn't go into the same version unless you override git tags. See links at the bottom.

**Comment by corest** [comment]
Switchi to tollerate all taints

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,12 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 ## [Unreleased]
 
+## [0.5.0] 2019-07-17
```
Comment (kopiczko) [inline CHANGELOG.md]: You also need to change https://github.com/giantswarm/kubernetes-node-exporter/blob/eb7b687f36463e1508c6f39614bec715bfe3bb58/helm/kubernetes-node-exporter-chart/Chart.yaml#L5

Comment (corest) [reply CHANGELOG.md]: changed but not commited

**Pull request merged**